src/Umbraco.Examine.Lucene/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -38,7 +39,11 @@ public static IUmbracoBuilder AddExamineIndexes(this IUmbracoBuilder umbracoBuil
                     s,
                     new object[]
                     {
-                        new DirectoryInfo(tempDir), s.GetRequiredService<IApplicationRoot>().ApplicationRoot
+                        new DirectoryInfo(tempDir),
+                        s.GetRequiredService<IApplicationRoot>().ApplicationRoot,
+                        s.GetRequiredService<ILockFactory>(),
+                        s.GetRequiredService<ILoggerFactory>(),
+                        true,
```

Comment:
That should be configurable, as this flag might affect general performance of site if any of indexes gets corrupted.

Reply:
Maybe so, but in its current state an application is stuck in a faulty state so I believe this should be on by default.
The 13.5 release with update of Examine actually changes nothing since the highly wanted fix is disabled.

Umbraco.CMS.Examine.EnableRecovery (default on) or any suggestions?

Reply:
@kows I am happy now with it :)